cmd/go-filecoin/util_test.go

```diff
@@ -18,7 +18,7 @@ func TestOptionalAddr(t *testing.T) {
 
 		opts := make(cmdkit.OptMap)
 
-		specifiedAddr, err := address.NewActorAddress([]byte("a new test address"))
+		specifiedAddr, err := address.NewSecp256k1Address([]byte("a new test address"))
```

Comment:
Actor addresses are now always converted to Id addresses before processing, so Secp256k1 addresses are safer for testing.